tests/installation/bootloader_start.pm

```diff
@@ -63,6 +63,11 @@ sub run {
         # As normally we also need `bootloader` for this scenario
         return if get_var('MEDIACHECK');
     }
+    # Prepare for agama.auto
```

Comment:
this code should be in `lib/bootloader_setup.pm`

Reply:
Updated. VR is running.